<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectdim</class>
 <widget class="QMainWindow" name="selectdim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
background-image: url(:/1./Watch_Dogs_2_Wallpapers_Backgrounds_2.0.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dimok">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>660</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Pixeled&quot;;
color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dimcancel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>660</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Pixeled&quot;;
color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:12pt &quot;Pixeled&quot;;
color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Select the Dimension of the LED Box&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_project_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retrybuttondim">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>660</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Pixeled&quot;;
color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string>Retry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Front View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twodim_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>650</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);
font: 63 18pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(95, 95, 95);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="twodim_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>650</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);
font: 63 18pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(95, 95, 95);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="twodim_8">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>650</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);
font: 63 18pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(95, 95, 95);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>twodim_8</zorder>
   <zorder>twodim_7</zorder>
   <zorder>twodim_6</zorder>
   <zorder>dimok</zorder>
   <zorder>dimcancel</zorder>
   <zorder>label_2</zorder>
   <zorder>image_project_label</zorder>
   <zorder>retrybuttondim</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1219</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ledbox.qrc"/>
 </resources>
 <connections/>
</ui>
